pkg/apis/agones/v1/gameserver.go

```diff
@@ -198,6 +202,15 @@ type PlayersSpec struct {
 	InitialCapacity int64 `json:"initialCapacity,omitempty"`
 }
 
+// CountersSpec tracks if counter specified (for giving error message if feature gate not set)
+type CountersSpec struct {
+}
```

Comment:
`Default` and `Capacity` ?

Reply:
The default for count is applied by the CRD automatically if no value for count is given, so default is not a settable but the count (which automatically overrides the default) is. 

As far as what to include is this struct -- is there a reason why capacity is needed here if the CRD already validates that it's an integer >= 0?

Reply:
If there is no values to store `Default` or `Count` user set values into, then you can't access that. The incoming and outgoing data for the CRD need somewhere to serialise and deserialise to and from to.